mbed_targets/_internal/target_attributes.py

```diff
@@ -82,24 +82,25 @@ def _read_json_file(path_to_file: pathlib.Path) -> Any:
 
 
 def _extract_target_attributes(all_targets_data: Dict[str, Any], target_name: str) -> Any:
-    """Extracts the attributes for a particular target from the targets data.
+    """Extracts the definition for a particular target from all the targets in targets.json.
 
     Args:
         all_targets_data: a dictionary representation of targets.json data, still
-        containing all the hierarchy structure.
-        target_name: the name of the target also known as 'board_type' in the online database.
+        containing all the un-flattened hierarchy structure.
```

Comment:
Maybe something like `raw targets.json data` would be a clearer description of the first argument? The fact it's a dict is clear because of typing, _un-flattened hierarchy structure_ I find hard to consume.

Reply:
c979061